### PR TITLE
Improvements to MENU.md

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Improvements include:
   1. added items
   2. clarified items
   3. corrected spelling errors
   4. corrected grammar errors
   5. improved consistency between items

### CODE REVIEW

**Comment by M66B** [comment]
I am glad somebody finally improves my English!
(I am not a native speaker)

**Comment by Gitoffthelawn** [comment]
Glad to help.  :)

File: MENU.md [hunk 1]
```diff
@@ -98,23 +100,23 @@
...
+* On/Off toggle- this will toggle restrictions for the selected app on/off without deleting the restriction settings
+* On demand ? - this will toggle on demand restrictions for the selected app
 * Category drill down - this will show or hide the individual functions of a category
 * Category info icon - this will open the category restrcions info on GitHub in a browser
-* Function book icon - this will show additional information about the function, possibly with a link to the google documentaion
-* Category restriction check mark - this will restrict or unrestrict the category for the currently selected app
-* Category on demand check mark - this will enable or disalbe on demand popups to the functions in the cateogry for the currently selected app
-* Function restrction check mark - this will restrict or unrestrict the indiviual function
-* Functions on demand check mark - this will enable or disalbe on demand popups for the individual function
+* Function book icon - this will show additional information about the function, possibly with a link to the Google documentaion
```
Comment (pylerSM) [inline MENU.md]: "documentaion"

Comment (Gitoffthelawn) [inline MENU.md]: Thanks... fixed.

**Pull request merged**